### DISCUSSION TITLE
Similar cool idea for randomizing audio elements inside audio clips.

### DISCUSSION THREAD

**Discussion by justevanjust** [kernicPanel/bitwig-randomizer#10] [Ideas]
I really like your bitwig-randomizer extension, and I have a similar cool idea for randomizing audio elements inside audio clips/blocks, but I'm not a programmer. Perhaps you know how this can be done for Bitwig Studio. How can I contact you?

**Comment by kernicPanel** [comment]
Hi! Thanks for your appreciation.
I didn't look for this kind of things in the API yet.
Maybe you can explain here what you have in mind?

**Comment by justevanjust** [comment]
I mean the randomization of the content of audio clips. Let's say I used the bass loop on audio track and then I want to quickly replace it with another bass loop, or any other loop. Bitwig has such a function in Project Panel -> Files -> Replace button , but it has not been finalized and instead of calling the bitwig browser, it calls windows explorer. If necessary, I could record a detailed video of exactly what I mean. Thanks!

**Comment by kernicPanel** [comment]
For now, I don't see how to do this, but I like the idea.
I'll leave this issue open, maybe I'll find something in the future.
